utils/gettab.f90

```diff
@@ -0,0 +1,28 @@
+!> @file
+!> @brief Read BUFR file containing embedded DX tables,
+!> and print the table to stdout
+
+program gettab
```

Comment:
consider renaming to get_table, or gettable or something like that.

Reply:
I would like to rename this to something that makes sense.  Also all the cryptic functions and subroutines.
 However, I am not the one who wrote this utility and it is under circulation for decades without being version controlled or packaged with the library.
I am trying to bring these orphans into one roof (or repo!) to help users (and developers) alike.

Reply:
this should be `get_table`, ideally.

Reply:
maybe 'bfdump -table'.  see comment for readmp.